packages/sdk/src/evm/common/currency.ts

```diff
@@ -192,3 +192,22 @@ export async function normalizeAmount(
   const decimals = await contractWrapper.readContract.decimals();
   return utils.parseUnits(AmountSchema.parse(amount), decimals);
 }
+
+export function toEther(amount: Amount): string {
+  return utils.formatEther(AmountSchema.parse(amount));
+}
+
+export function toWei(amount: BigNumberish): BigNumber {
+  return utils.parseEther(AmountSchema.parse(amount));
```

Comment:
Here you do y have to parse anything, just pass the BigNumberish value in

Reply:
"pasreEther" params accept string only `parseEther(ether: string)`

Reply:
right so it should be the opposite then:

`toEther(amount: BigNumberish)`
and
`toWei(amount: Amount)`